frontend/src/pages/RunList.test.tsx

```diff
@@ -434,7 +439,75 @@ describe('RunList', () => {
     tree = shallow(<RunList {...props} />);
     await (tree.instance() as RunListTest)._loadRuns({});
     expect(props.onError).not.toHaveBeenCalled();
-    expect(tree).toMatchSnapshot();
+    expect(tree).toMatchInlineSnapshot(`
```

Comment:
Why did you change this to be an inline snapshot?

Reply:
I am following the testing best practices in https://github.com/kubeflow/pipelines/pull/3166, which allows us to inspect the UI element outcome on the spot. But feel free to argue differently. :)